assets/src/web-stories-block/block/block-types/selected-stories/edit.js

```diff
@@ -18,74 +18,69 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { ThemeProvider } from 'styled-components';
-import {
-  theme as externalDesignSystemTheme,
-  lightMode,
-} from '@web-stories-wp/design-system';
 
 /**
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { useState, useEffect } from '@wordpress/element';
+import { useState, useEffect, useCallback } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import StoriesInspectorControls from '../../components/storiesInspectorControls';
-import StoriesBlockApiProvider from '../../api/apiProvider';
 import StoriesPreview from '../../components/storiesPreview';
 import EmbedPlaceholder from './embedPlaceholder';
 import FetchSelectedStories from './fetchSelectedStories';
 
-const SelectedStoriesEdit = ({
+function SelectedStoriesEdit({
```

Comment:
Why change this?

Reply:
Just updated for consistency while I was revamping the component. We usually write components this way.